src/Http/BaseApplication.php

```diff
@@ -188,6 +189,7 @@ public function bootstrap(): void
         // phpcs:ignore
         $plugins = @include $this->configDir . 'plugins.php';
         if (is_array($plugins)) {
+            $this->plugins->setContainer($this->getContainer());
```

Comment:
These 2 places where the least intrusive places where I was able to set the container in the plugin collection without other core tests failing

Reply:
Yeah this makes sense as we are setting up the framework so it can run the application.

Reply:
The container instance is being set for the plugin collection only so that it can be passed on to the plugins. But plugins already gets access to the container through the app instance that's passed to `bootstrap()` and through the `services()` hook. So I don't any reason for increasing the API surface.